src/main/rx/rx.c

```diff
@@ -468,18 +468,123 @@ void rxSetUplinkTxPwrMw(uint16_t uplinkTxPwrMwValue)
 }
 #endif
 
+#if defined(USE_PWM) || defined(USE_PPM)
```

Comment:
did these methods change, or were they just moved?  if they were moved can they be moved back?  if they were moved and modified can they be moved back and made to work in their old position in the file.

Moving methods around in files and renaming things, struct members, method names, etc, is *always* best done as a separate commit, and ideally as the first commit of a PR, so that reviewers can look at the commit list and ignore the move/rename methods and just focus on the main changes.

Reply:
With regards to my second point, as an example of me doing exactly this, yesterday, is here:  https://github.com/betaflight/betaflight/pull/11456/commits  The second commit is atomic and just replaces 1 method and a boolean argument with two different methods.  It makes review much easier as reviewers can focus on the other commits.